Bill3.java

```diff
@@ -2,7 +2,12 @@
 public class Bill3 {
 	public static void main(String[] args) {
 		// To get you started, here is the first line in the program:
-	    String name1 = args[0];
-	    // Replace this comment with the rest of your code   
+	    String name1 = (args[0]);
+		String name2= (args[1]);
+		String name3= (args [2]);
+		int value= Integer.parseInt(args[3]);
+       double a=(double) value/3.0;
```

Comment:
This is not a meaningful variable name...
When I am reading this, how can I know what this variable represents?